docs/pages/link.js

```diff
@@ -222,11 +106,40 @@ function TabExample() {
     </Box>
   );
 }`}
-      />
-      <Example
-        id="Permutations"
-        name="Permutations: tapStyle and hoverStyle"
-        defaultCode={`
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          title="Link and Text"
+          description="Use Link within [text](https://gestalt.pinterest.systems/text) to get the correct font and underline color."
```

Comment:
If we're referring to our Text component, it should be capitalized

Reply:
DONE